.github/workflows/qemu-jobs.yml

```diff
@@ -0,0 +1,193 @@
+name: Build and check tests for QFlex enabled and base QEMU
+
+on: 
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+  workflow_dispatch:
+
+jobs:
+  build_and_check_job_definition: # native_build_job_definition:
+    runs-on: ubuntu-latest
+    name: Configure, build and check QEMU # native_build_job_definition
+    strategy:
+      matrix:
+        include:
+          - # NOTE: --enable-fdt=system --enable-slirp=system was removed
+            build-name: Build for Ubuntu 20.04 + check # build-system-ubuntu
+            targets: aarch64-softmmu
+            make-check-args: check
+          - build-name: Build for user-mode + check-tcg # build-user
+            targets: aarch64-linux-user
+            configure-args: --disable-tools --disable-system
+            make-check-args: check-tcg
+          - build-name: Build for system-mode + check-tcg # build-some-softmmu
+            targets: aarch64-softmmu
+            configure-args: --disable-tools --enable-debug
+            make-check-args: check-tcg
+#          - build-name: build-user-plugins
+#            targets: aarch64-linux-user
+#            configure-args: --disable-tools --disable-system --enable-plugins --enable-debug-tcg
+#            make-check-args: check-tcg
+          - build-name: Build for system-mode with plugins enabled + check-tcg # build-some-softmmu-plugins
+            targets: aarch64-softmmu
+            configure-args: --disable-tools --disable-user --enable-plugins --enable-debug-tcg
+            make-check-args: check-tcg
+          - build-name: Build using clang + check-tcg, check-qtest # clang-system
+            targets: aarch64-softmmu
+            configure-args: --cc=clang --cxx=clang++
+              --extra-cflags=-fsanitize=undefined --extra-cflags=-fno-sanitize-recover=undefined
+            make-check-args: check-qtest check-tcg
+#          - build-name: clang-user
+#            targets: aarch64-linux-user
+#            configure-args: --cc=clang --cxx=clang++ --disable-system
+#              --extra-cflags=-fsanitize=undefined --extra-cflags=-fno-sanitize-recover=undefined
+#            make-check-args: check-unit check-tcg
+          - build-name: Build with TCG interpreter, lauches boot and cdrom tests # build-tci
```

Comment:
Same here. No tests actually run.

I compared this job to the first vanilla build (just launching check) - and that configuration actually ran the boot-serial test and CD-ROM test. Looks like the problem here is with `enable-tcg-interpreter`. Why do we need this test? Can you explain?

Reply:
From Resolved in https://github.com/parsa-epfl/qemu/pull/62#issuecomment-882524912:
I though originally this was the only tests to launch boot-serial and CD-ROM tests, and thus I enabled it. Given that my understanding was wrong, TCI is actually a very niche feature that doesn't seem to really been maintained since 2011 and with little documentation.
From my understanding TCI is there to execute TCG binary code generated by QEMU, similarly to Java, TCI would stand as the JVM and Java Code as an intermediate representation of TCG binary.
It is useful to support very nieche host platforms such as those not listed here (arm, i386, ia64, ppc, ppc64, s390, sparc, x86_64).
I'll remove this test.